test/integration/service/serverspec/debian_spec.rb

```diff
@@ -43,6 +43,10 @@
         it { should be_grouped_into 'root' }
       end
 
+      describe file('/etc/service/plain-defaults/log/config') do
```

Comment:
Thanks for pointing this out. I think it's okay for the config file to live in the same directory as the log's run script, as long as we also symlink it into the directory where the log process will write to. I'll update the PR to ensure that linking occurs.